tutorials/math/matrices_and_transforms.rst

```diff
@@ -284,13 +284,15 @@ Shearing the transformation matrix (advanced)
           explores an uncommonly used aspect of transformation matrices
           for the purpose of building an understanding of them.
 
+          Since Godot 3.2.2, Node2D provides a shearing property out of the box.
```

Comment:
I reverted it, so it's no longer true. I might add it back in 3.2.3 if we find a fix for the issue it introduced.

Reply:
Please remove the reference to 3.2.2. It hasn't been cherry-picked back yet, so it might not make it to 3.2.x any time soon.